<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>object_setting</class>
 <widget class="QWidget" name="object_setting">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>192</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1260</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string notr="true">object_setting</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(3, 3, 13);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="top_logo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>202</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>202</width>
           <height>32</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Sans</family>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resourece.qrc">:/ui/ui/images/logo.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="shutdown_bnt">
           <property name="minimumSize">
            <size>
             <width>61</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>61</width>
             <height>31</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>NanumSquareRound</family>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(237, 51, 59);
color: rgb(255, 255, 255);
border-radius: 15px;
</string>
           </property>
           <property name="text">
            <string>닫기</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="person_label">
              <property name="minimumSize">
               <size>
                <width>203</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>203</width>
                <height>28</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Sans</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(179,179,179);
background-color: rgba(255, 255, 255, 0);</string>
              </property>
              <property name="text">
               <string>사람</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="person_active_bnt">
              <property name="minimumSize">
               <size>
                <width>61</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>61</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    padding: 0px; /* 패딩도 0으로 설정 */
    outline: none;
}

/* hover, pressed 상태에도 아무 효과 없음 */
QPushButton:hover {
    background-color: transparent;
    border: none;
}

QPushButton:pressed {
    background-color: transparent;
    border: none;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../resourece.qrc">
                <normaloff>:/ui/ui/images/icon-switch-off.png</normaloff>
                <normalon>:/ui/ui/images/icon-switch-on.png</normalon>:/ui/ui/images/icon-switch-off.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>55</width>
                <height>103</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="person_conf_slider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>114</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>31</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QSlider::handle:horizontal {
    background-color: rgb(192, 191, 188);
    border: 1px solid #5c5c5c; /* 핸들의 테두리 */
    width: 20px; /* 핸들의 너비 */
    height: 5px; /* 핸들의 너비 */
     margin: -4px 0px; /* 핸들의 위, 아래 여백 */
    border-radius: 4px; /* 핸들의 모서리 둥글게 */
}

QSlider::groove:horizontal {
    border: 1px solid #999999;
    height: 5px; /* 슬라이더의 트랙 높이 */
    background: rgb(100, 100, 100); /* 슬라이더 트랙의 기본 색상 */
    border-radius: 4px;
    margin: 0px; /* 트랙의 여백을 없앰 */
}

QSlider::sub-page:horizontal {
    background: rgb(30, 195, 55); /* 게이지 색상 */
    border: 1px solid #777;
    height: 5px; /* 트랙과 동일한 높이로 설정 */
    border-radius: 4px;
    margin: 0px; /* 게이지 여백을 없앰 */
}
</string>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="singleStep">
               <number>2</number>
              </property>
              <property name="value">
               <number>33</number>
              </property>
              <property name="tracking">
               <bool>false</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="person_conf_value">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>31</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">
    QSpinBox {
        subcontrol-origin: padding;
        subcontrol-position: top right;
        background: rgb(13, 16, 23);
        color: rgb(255, 255, 255);

    }</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>33</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="car_label">
              <property name="minimumSize">
               <size>
                <width>203</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>203</width>
                <height>28</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Sans</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(179,179,179);
background-color: rgba(255, 255, 255, 0);</string>
              </property>
              <property name="text">
               <string>탈것(자동차, 자전거, 오토바이)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="car_active_bnt">
              <property name="minimumSize">
               <size>
                <width>61</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>61</width>
                <height>25</height>
               </size>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    padding: 0px; /* 패딩도 0으로 설정 */
    outline: none;
}

/* hover, pressed 상태에도 아무 효과 없음 */
QPushButton:hover {
    background-color: transparent;
    border: none;
}

QPushButton:pressed {
    background-color: transparent;
    border: none;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../resourece.qrc">
                <normaloff>:/ui/ui/images/icon-switch-off.png</normaloff>
                <normalon>:/ui/ui/images/icon-switch-on.png</normalon>:/ui/ui/images/icon-switch-off.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>55</width>
                <height>103</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="car_conf_slider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>114</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>31</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QSlider::handle:horizontal {
    background-color: rgb(192, 191, 188);
    border: 1px solid #5c5c5c; /* 핸들의 테두리 */
    width: 20px; /* 핸들의 너비 */
    height: 5px; /* 핸들의 너비 */
     margin: -4px 0px; /* 핸들의 위, 아래 여백 */
    border-radius: 4px; /* 핸들의 모서리 둥글게 */
}

QSlider::groove:horizontal {
    border: 1px solid #999999;
    height: 5px; /* 슬라이더의 트랙 높이 */
    background: rgb(100, 100, 100); /* 슬라이더 트랙의 기본 색상 */
    border-radius: 4px;
    margin: 0px; /* 트랙의 여백을 없앰 */
}

QSlider::sub-page:horizontal {
    background: rgb(30, 195, 55); /* 게이지 색상 */
    border: 1px solid #777;
    height: 5px; /* 트랙과 동일한 높이로 설정 */
    border-radius: 4px;
    margin: 0px; /* 게이지 여백을 없앰 */
}
</string>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="singleStep">
               <number>2</number>
              </property>
              <property name="value">
               <number>33</number>
              </property>
              <property name="tracking">
               <bool>false</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="car_conf_value">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>31</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">
    QSpinBox {
        subcontrol-origin: padding;
        subcontrol-position: top right;
        background: rgb(13, 16, 23);
        color: rgb(255, 255, 255);

    }</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>33</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="fire_label">
              <property name="minimumSize">
               <size>
                <width>203</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>203</width>
                <height>28</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Sans</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(179,179,179);
background-color: rgba(255, 255, 255, 0);</string>
              </property>
              <property name="text">
               <string>불</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="fire_active_bnt">
              <property name="minimumSize">
               <size>
                <width>61</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>61</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
    padding: 0px; /* 패딩도 0으로 설정 */
    outline: none;
}

/* hover, pressed 상태에도 아무 효과 없음 */
QPushButton:hover {
    background-color: transparent;
    border: none;
}

QPushButton:pressed {
    background-color: transparent;
    border: none;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../resourece.qrc">
                <normaloff>:/ui/ui/images/icon-switch-off.png</normaloff>
                <normalon>:/ui/ui/images/icon-switch-on.png</normalon>:/ui/ui/images/icon-switch-off.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>55</width>
                <height>103</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="fire_conf_slider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>114</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>31</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QSlider::handle:horizontal {
    background-color: rgb(192, 191, 188);
    border: 1px solid #5c5c5c; /* 핸들의 테두리 */
    width: 20px; /* 핸들의 너비 */
    height: 5px; /* 핸들의 너비 */
     margin: -4px 0px; /* 핸들의 위, 아래 여백 */
    border-radius: 4px; /* 핸들의 모서리 둥글게 */
}

QSlider::groove:horizontal {
    border: 1px solid #999999;
    height: 5px; /* 슬라이더의 트랙 높이 */
    background: rgb(100, 100, 100); /* 슬라이더 트랙의 기본 색상 */
    border-radius: 4px;
    margin: 0px; /* 트랙의 여백을 없앰 */
}

QSlider::sub-page:horizontal {
    background: rgb(30, 195, 55); /* 게이지 색상 */
    border: 1px solid #777;
    height: 5px; /* 트랙과 동일한 높이로 설정 */
    border-radius: 4px;
    margin: 0px; /* 게이지 여백을 없앰 */
}
</string>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="singleStep">
               <number>2</number>
              </property>
              <property name="value">
               <number>33</number>
              </property>
              <property name="tracking">
               <bool>false</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="fire_conf_value">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>31</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">
    QSpinBox {
        subcontrol-origin: padding;
        subcontrol-position: top right;
        background: rgb(13, 16, 23);
        color: rgb(255, 255, 255);

    }</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>33</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resourece.qrc"/>
 </resources>
 <connections/>
</ui>
